src/egithub.erl

```diff
@@ -901,24 +955,50 @@ maybe_queue_request(Cred, Url, JsonBody, Options) ->
             egithub_req:queue(Cred, Url, post, JsonBody)
     end.
 
-maybe_append_qs_params(issues, Url, Opts) ->
-    Params = #{filter    => maps:get(filter, Opts, "assigned"),
-               state     => maps:get(state, Opts, "open"),
-               labels    => maps:get(labels, Opts, ""),
-               sort      => maps:get(sort, Opts, "created"),
-               direction => maps:get(direction, Opts, "asc"),
-               since     => maps:get(since, Opts, "")},
-    case maps:size(Opts) > 0 of
+maybe_append_qs_params(Url, Params) ->
+    case maps:size(Params) > 0 of
         false ->
             io_lib:format(Url, []);
         true ->
-            QS = maps:fold(fun (_K, "", Acc) ->
-                                   Acc;
-                               (K, V, Acc) ->
-                                   [io_lib:format("~s=~s", [K, V]) | Acc]
-                           end, [], Params),
-            io_lib:format("~s?~s", [Url, string:join(lists:reverse(QS), "&")])
+            QS = format_qs_params(maps:to_list(Params), []),
+            [lists:flatten(io_lib:format("~s?~s", [Url, QS]))]
     end.
+format_qs_params([], Acc) ->
```

Comment:
Would you mind adding an empty line before this one, considering this is a new function / different from the one above?